lib/engines/content_object_store/app/controllers/content_object_store/content_block_editions_controller.rb

```diff
@@ -16,6 +16,14 @@ def create
     @schema = ContentObjectStore::SchemaService.schema_for_block_type(root_params[:block_type])
     @content_block_edition = ContentObjectStore::ContentBlockEdition.create!(edition_params)
 
+    result = Services.publishing_api.put_content(@content_block_edition.document.content_id, {
```

Comment:
I was going to say I'd expect this call to be in a service or a callback, but I've seen you've done this in the next commit